client/lib/plugins/wporg-data/curated.json

```diff
@@ -0,0 +1,72 @@
+[
+	{
+		"name": "WooCommerce",
+		"author": "<a href=\"https://woocommerce.com\">Automattic</a>",
+		"rating": 92,
```

Comment:
I don't believe we should be hard coding rating here since this will change more rapidly than an icon or slug. Can we show a placeholder for the rating until we can fetch it? This was partly why I suggested a new component.